### PR TITLE
Rainy day#108

### PR DESCRIPTION
**Pull request by bendelonlee**
Adds a RainyDay class that generates rainy days based on user phone numbers in the database where their gardens have an over 50% chance of precipitation. 
Adds a RainyDayTexter class that handles calls to Twillio and composes texts.
Renames RainyDayTexterJob to RainyDayJob.

### CODE REVIEW

File: spec/jobs/rainy_day_job_spec.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  end
+
+  it 'reschedules itself for the next morning', :vcr do
+    expect {
+      RainyDayJob.perform_now
+    }.to have_enqueued_job(RainyDayJob).at(RainyDayJob.early_next_morning)
+  end
+
+  it 'sends texts', :vcr do
+    def weather_service_stub(chance)
```
Comment (asmolentzov) [inline spec/jobs/rainy_day_job_spec.rb]: yay!

File: spec/texters/rainy_day_texter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+describe RainyDayTexter do
+  describe 'class methods' do
+
+    it 'rainy_day_text' do
+      garden = create(:garden)
+      chance = 80
+
+      expected_text = {
```
Comment (abenetka) [inline spec/texters/rainy_day_texter_spec.rb]: Fancy

File: spec/models/rainy_day_spec.rb [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      user_1 = create(:user)
+      create(:garden, user: user_1)
+      user_2 = create(:user, telephone: '23482398')
+      garden_1 = create(:garden, user: user_2, zip_code: "80000")
+      garden_2 = create(:garden, user: user_2, zip_code: "80125")
+      garden_3 = create(:garden, user: user_2, zip_code: "80125")
+      expect(RainyDay.zip_codes).to eq([garden_1.zip_code, garden_2.zip_code])
+    end
+    it 'returns all users whose gardens will be having a rainy day (50% or greater rain chance)' do
+
```
Comment (asmolentzov) [inline spec/models/rainy_day_spec.rb]: Nothing in this test?

File: spec/models/rainy_day_spec.rb [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    it 'zip_codes' do
+      user_1 = create(:user)
+      create(:garden, user: user_1)
+      user_2 = create(:user, telephone: '23482398')
+      garden_1 = create(:garden, user: user_2, zip_code: "80000")
+      garden_2 = create(:garden, user: user_2, zip_code: "80125")
+      garden_3 = create(:garden, user: user_2, zip_code: "80125")
+      expect(RainyDay.zip_codes).to eq([garden_1.zip_code, garden_2.zip_code])
+    end
+    it 'returns all users whose gardens will be having a rainy day (50% or greater rain chance)' do
```
Comment (abenetka) [inline spec/models/rainy_day_spec.rb]: Is this test going to be filled in?

**Pull request merged**